src/plugins/vis_type_timeseries/public/application/visualizations/views/timeseries/index.js

```diff
@@ -139,6 +140,7 @@ export const TimeSeries = ({
           type: tooltipMode === 'show_focused' ? TooltipType.Follow : TooltipType.VerticalCursor,
           headerFormatter: tooltipFormatter,
         }}
+        externalPointerEvents={{ tooltip: { visible: true } }}
```

Comment:
This option now only controls the tooltip visibility. The crosshair is always shown when using `dispatchExternalPointerEvent`.

```suggestion
        externalPointerEvents={{ tooltip: { visible: false } }}
```